client/site-profiler/components/site-profiler-v2.tsx

```diff
@@ -148,14 +151,12 @@ export default function SiteProfilerV2( props: Props ) {
 						) }
 						width="medium"
 					>
-						{ showBasicMetrics && (
-							<ResultsHeader
-								domain={ domain }
-								performanceCategory={ performanceCategory }
-								isWpCom={ isWpCom }
-								onGetReport={ () => setIsGetReportFormOpen( true ) }
-							/>
-						) }
+						<ResultsHeader
```

Comment:
After refreshing the page after the initial load on the report URL, the audits are not rendered. 

![CleanShot 2024-06-06 at 11 29 40@2x](https://github.com/Automattic/wp-calypso/assets/86406124/b1dd147b-a941-472f-85a2-6b841ec959a3)

![CleanShot 2024-06-06 at 11 29 32@2x](https://github.com/Automattic/wp-calypso/assets/86406124/730c048b-8c78-48ab-95d0-c867cffaf7bb)

Reply:
> After refreshing the page after the initial load on the report URL, the audits are not rendered.

That is a consequence of not calling the basic endpoint if the hash is present. That should be fixed by the following task https://github.com/Automattic/dotcom-forge/issues/7579 or alternatively we can use these changes https://github.com/Automattic/wp-calypso/pull/91548 to call the basic endpoint without the `advance` param. 

What do you prefer? I would go for both approaches to be explicit.

Reply:
> That should be fixed by the following task https://github.com/Automattic/dotcom-forge/issues/7579

I would prefer this.